src/transformers/models/whisper/modeling_tf_whisper.py

```diff
@@ -1324,6 +1327,228 @@ def call(
             encoder_attentions=outputs.encoder_attentions,
         )
 
+    def generate(
+        self,
+        inputs: Optional[tf.Tensor] = None,
+        generation_config: Optional[GenerationConfig] = None,
+        logits_processor: Optional[TFLogitsProcessorList] = None,
+        seed=None,
+        return_timestamps=None,
+        task=None,
+        language=None,
+        is_multilingual=None,
```

Comment:
```suggestion
        seed: Optional[List[int]] = None,
        return_timestamps: Optional[bool] = None,
        task: Optional[str] = None,
        language: Optional[str] = None,
        is_multilingual: Optional[bool] = None,
```

Reply:
Thanks. I know - I copied directly from our torch code, me too lazy. Will keep torch code the untouched in this PR though.